app/src/main/res/values/strings.xml

```diff
@@ -533,36 +533,30 @@
-        <item>Can I alter the sequence of pages in a PDF?#####Yes, to change the sequence of pages in the PDF, select Reorder Pages from the home page. This feature can also be accessed from the options menu by swiping from left to right or clicking the 3 horizontal lines at the top left corner of the app.\nYou can use the arrow keys to order the pages or even delete one or more pages.</item>
-        <item>Can I extract images or text from a PDF?#####Yes, to extract images or text from a PDF, use Extract Images and Extract Text available on the home page or in the options menu which appears if you swipe from left to right or click on the 3 horizontal lines at the top left of the app.</item>
-        <item>Before converting a text file into a PDF, can I specify enhancement options?#####Yes, you can use the enhancement options available in the Text to PDF option.</item>
-        <item>Can I convert a PDF into different file formats such as PPT, PPTX, ODT, DOC, DOCX?#####No, the PDF can only be converted to text format at the moment but we are planning to add other conversion options soon!</item>
-        <item>Is Images-to-pdf software available to any other operating system?#####There is no intention to migrate this software to another operating system.</item>
-        <item>Where can I download Images-to-pdf software?#####The software can be downloaded from Google PlayStore and also on F-droid</item>
-        <item>Can I contribute to Images-to-pdf project?#####Yes, please! \nFeel free to share your ideas(feature requests), submit a bug or pull request for anything you feel should be added. \nThis project currently has multiple contributors that add as little as a single line change. This project is opened to contributions of any kind. You can also fork the project and further develop it on your own so long as you work within the confines of the license. Pull requests of your contributions will be merged if they fulfill some certain requirements. Read the project\'s Contributor\'s guide to get started on how to contribute.</item>
-        <item>I have some questions not contained in this FAQs document, how can I contact the developer?#####For further inquiries, you can contact the developer via email. Send an email to swati4star@gmail.com.</item>
+        <item>What image file formats are supported?#####The app supports jpeg, jpg, tiff, gif, psd, bmp, eps, png, etc.</item>
+        <item>Can I create a new text file using the app?#####You can only create a new text file by extracting texts from an existing PDFs. To create a text file, \n1) Click the <strong>Extract Text</strong> option on the homepage. \n2) Select the PDF using <strong>SELECT PDF FILE<strong> button. \n3) Then, click <strong>EXTARCT TEXT<strong>. \n4) Once done, a text file with the texts from the PDF file will be created.</item>
```

Comment:
let's make it consistent.
1. All caps + strong
2. Strong

Choose any one the above & follow it throughout the FAQs

Reply:
I've used caps because the button/element name in the app is in caps.